apiserver/facades/client/machinemanager/machinemanager_test.go

```diff
@@ -1648,9 +1648,8 @@ type IsBaseLessThanMachineManagerSuite struct{}
 // but complex enough to warrant being exported from an export test package for
 // testing.
 func (s *IsBaseLessThanMachineManagerSuite) TestIsBaseLessThan(c *gc.C) {
-	workloadVersions, err := corebase.AllWorkloadVersions()
-	c.Assert(err, jc.ErrorIsNil)
-	vers := workloadVersions.Values()
+	workloadBases := corebase.WorkloadBases()
+	vers := transform.Slice(workloadBases, func(b corebase.Base) string { return b.Channel.Track })
```

Comment:
Could pass it here without assigning a variable.

Reply:
Done